<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_appendDialog</class>
 <widget class="QDialog" name="_appendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="lineEd_sta">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEd_end">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEd_per">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEd_offt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEd_arrt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEd_model">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEd_price">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NUM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_start">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_end">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>END</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_period">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>PERIOD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_offt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OFFT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_arrt">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ARRT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_model">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MODEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PRICE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>20</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Append!</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>80</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEd_num">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
